packages/orbit-design-tokens/output/deprecated-tokens.json

```diff
@@ -1947,6 +1947,56 @@
     "category": "Breakpoints",
     "version": "6.0.0"
   },
+  "elevationActionBackground": {
+    "replacement": "paletteWhiteNormal",
+    "category": "Colors",
+    "version": "9.0.0"
+  },
+  "elevationActionBoxShadow": {
```

Comment:
I have just realized we now have two levels of deprecations 🤦‍♂️ 
The token `boxShadowAction` is deprecated and suggests `elevationActionBoxShadow` as replacement. But now `elevationActionBoxShadow` is also deprecated and suggests `elevationLevel1BoxShadow` as replacement.

I am not entirely sure about how this will impact the breaking change, but for instance, I see the `boxShadowAction` (the _double deprecated_ one) is currently being used [here](https://github.com/kiwicom/orbit/blob/marco/shadow-tokens/packages/orbit-tailwind-preset/src/index.ts#L256)… 😬  The same for `modal-scrolled` and `modal` defined right below: they're also using doubly deprecated tokens.

So, yes, I honestly don't know what to do with these. But I'd recommend going through all the tokens (even the older ones) and replace them for the new ones. For example, in that case, replace that `switch` definition by the correct `elevationLevel1BoxShadow` token (although I'm also not sure about the consequences of such change…)

I did the quick exercise of completely deleting the tokens and the only place where the build failed was the file mentioned above, but I am not 100% sure this is all the confidence we need.